edx_rest_framework_extensions/authn/README.rst

```diff
@@ -0,0 +1,8 @@
+This directory will contain all authentication related extensions.
+
+It contains the following subdirectory:
```

Comment:
Nit: No need to document self-evident subdirectories.

Reply:
We can update in the next PR.